File: Models/OrderDto.cs
namespace Atividade.Api.Models;

public class OrderDto
{
    public int StoreId { get; set; }
    public string ConsumerName { get; set; } = string.Empty;
    public double Price { get; set;}
    public DateTime DateCreated { get; set; }
    public List<int> ItemIds { get; set; } = default!;
    public List<int> ServiceIds = default!;
}

File: ClientApp/src/app/components/dashboard/models/OrderDto.ts
import { AdditionalServiceDto } from "./AdditionalServiceDto";
import { ItemDto } from "./ItemDto";
import { StatusDto } from "./StatusDto";

export class OrderDto {
  consumerName?: string;
  price: number = 0;
  date: Date = new Date();
  itemIds!:number[];
  additionalServiceIds!: number[];
}
